distributed/dashboard/scheduler.py

```diff
@@ -4,6 +4,7 @@
 import os.path
 import json
 import logging
+from time import time
```

Comment:
In Windows `time` returns integer values, which can be unpleasant.  

We sometimes use either `IOLoop.current().time()` or `distributed.metrics.time()`.  My guess is that these values are computed with `metrics.time` and so it's probably best to use that for cross-platform support.